tests/e2e/utils/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+## Added
+
+- `utils.waitForTimeout( delay )` pause processing for `delay` milliseconds
+- `AdminEdit`, `OrderEdit` classes with utility functions for the respctive edit screens
```

Comment:
Minor typo:

```suggestion
- `AdminEdit`, `OrderEdit` classes with utility functions for the respective edit screens
```

Reply:
Nice catch, fixed in https://github.com/woocommerce/woocommerce/pull/30696/commits/5ccf3a564fb4cc4cb4c392584320c6e6ec20069d